pom.xml

```diff
@@ -524,6 +524,23 @@
            </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>2.1.0</version>
+        <configuration>
+          <to>
+            <image>docker.io/plos/rhino:${project.version}</image>
+          </to>
+          <from>
+            <image>docker.io/plos/tomcat:latest</image>
```

Comment:
should we fix the version here so we don't get unexpected tomcat upgrades?

Reply:
The plos/tomcat image has the tomcat version fixed so this will only change if we change that image. It may be worth versioning at some point but I don't expect this to change very often anyway.